### PR TITLE
Add pip support

### PR DESCRIPTION
**Pull request by a-ignatyev**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -57,20 +57,13 @@ def find_version(*file_paths):
...
 
-DEPENDENCY_LINKS = []
-
 python_version = sys.version_info[:2]
-if python_version[0] < 3:
-    print("Only Python > 3.5 is supported")
-    sys.exit(0)
-elif python_version[1] < 6:
+if python_version[0] < 3 or python_version[1] < 6:
     print("Only Python > 3.6 is supported")
```
Comment (daniil-lyakhov) [inline setup.py:62]: You mean >= 3.6 ?

Comment (a-ignatyev) [reply setup.py]: Yes. Fixed.

File: setup.py [hunk 2]
```diff
@@ -57,20 +57,13 @@ def find_version(*file_paths):
...
-DEPENDENCY_LINKS = []
-
 python_version = sys.version_info[:2]
-if python_version[0] < 3:
-    print("Only Python > 3.5 is supported")
-    sys.exit(0)
-elif python_version[1] < 6:
-    print("Only Python > 3.6 is supported")
+if python_version[0] < 3 or python_version[1] < 6:
+    print("Only Python >= 3.6 is supported")
```
Comment (vshampor) [inline setup.py:62]: Our current README states support for >=3.7 only, should probably update this condition as well

**Review by vshampor — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (alexsu52) [reply setup.py:62]: What reason to bump up the python version? Tensorflow 2.4 requires python>=3.6 and PyTorch 1.7 too.

Comment (vshampor) [reply setup.py:62]: We actually validate on 3.7 only.

Comment (vshampor) [reply setup.py:62]: Ok, since TF requires 3.6, will postpone updating setup.py.

**Comment by vshampor** [comment]
Install build 253 passed.

**Review by vshampor — APPROVED** [review]

**Pull request merged**